oneflow/core/functional/impl/nn_functor.cpp

```diff
@@ -76,6 +76,9 @@ class ConvBaseFunctor {
                            const std::vector<int32_t>& padding,
                            const std::vector<int32_t>& dilation, const int32_t& groups,
                            const std::string& channel_pos) const {
+    CHECK_EQ_OR_RETURN(num_spatial_dims_ + 2, input->ndim())
+        << "Expected " << num_spatial_dims_ + 2 << "D input to conv" << num_spatial_dims_
+        << "d, but got input of size: " << input->shape()->ToString();
```

Comment:
差不多像这样，如果shape已经重载了`operator<<`的话，那么最后那个`ToString()`也不用了
```suggestion
        << fmt::format("Expected {}D input to conv{}d, but got input of size: {}", num_spatial_dims_ + 2, num_spatial_dims_, input->shape()->ToString());
```

Reply:
需要 `#include<fmt/core.h>`